chess/src/history.rs

```diff
@@ -59,7 +60,7 @@ impl History {
     }
 
     pub fn clone_current_state(&self) -> BoardState {
```

Comment:
i might just take this function out since it's `Copy` now

Reply:
Oh yeah I forgot about that